.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
+  
```

Comment:
- [ ] Please make sure to set up the EsLint linters correctly. Kindly update the `./github/workflows/linters.yml` file using [this](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) configuration from the [linters setup for HTML, CSS, and JavaScript](https://github.com/microverseinc/linters-config/tree/master/html-css-js) projects.

- This is a required change to use the correct linters setup. However, in the future, setting up the inters incorrectly can result in the invalidation of your submission.